README.md

```diff
@@ -60,6 +60,11 @@ An open source library for publishers and third party container solution provide
 - `aolhb.init()`  
    Library initialization method. It sends bid request for each placement based on configs passed as params.  
    Params: `bidRequestConfig`, `placementsConfigs`
+   
+- `aolhb.addNewAd()`  
```

Comment:
I feel as this method can be pretty commonly uses, so I would suggest that we add a code example for it.